packages/api/models/pdrs.js

```diff
@@ -1,113 +0,0 @@
-'use strict';
-
-const {
-  getMessageExecutionArn,
-} = require('@cumulus/message/Executions');
-const Logger = require('@cumulus/logger');
-const pvl = require('@cumulus/pvl');
-
-const Manager = require('./base');
-const pdrSchema = require('./schemas').pdr;
-
-const logger = new Logger({ sender: '@cumulus/api/models/pdrs' });
-
-class Pdr extends Manager {
-  constructor() {
-    super({
-      tableName: process.env.PdrsTable,
-      tableHash: { name: 'pdrName', type: 'S' },
-      schema: pdrSchema,
-    });
-  }
-
-  /**
-   * Generate PAN message
-   *
-   * @returns {string} the PAN message
-   */
-  static generatePAN() {
```

Comment:
Curious that this isn't being used anywhere.    Shouldn't stop this work, but I'm slightly concerned that I'm not sure if DAACs have just written their own PAN send/handling routine, or if it's just not being used....